common/scripts/get_image_sha.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright 2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Get the SHA from the RepoDigests section of an operand image.
+# If running on macOS, GNU sed is required. Use "brew install gnu-sed"
+# Do "docker login" before running this script.
+# Run this script from the parent dir by typing "scripts/get-image-sha.sh"
+
+FILE=deploy/operator.yaml
+
+# check the input parms
+REGISTRY=$1
+NAME=$2
+TAG=$3
+TYPE=$4
+if [[ $REGISTRY == "" ]] || [[ $NAME == "" ]] || [[ $TAG == "" ]] || [[ $TYPE == "" ]]
+then
+   echo "Missing param. Need image registry, image name, image tag, and env variable indicating operand type. Type will be OPERATOR for operator image as input"
+   echo "for eg: quay.io/opencloudio audit-policy-controller 3.5.0 POLICY_CTRL_TAG_OR_SHA"
+   exit 1
+fi
+
+# pull the image
+IMAGE="$REGISTRY/$NAME:$TAG"
+echo "Pulling image $IMAGE"
+docker pull "$IMAGE" &>/dev/null
+
+# get the SHA for the image
+DIGEST="$(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE" )"
```

Comment:
You might need to change this command to this: https://github.com/IBM/ibm-cert-manager-operator/blob/master/common/scripts/get_image_sha_digest.sh#L41
Reason:
The command used might result in different SHA (if you have multiple images from different registry available). You can either continue using the docker image inspect command by making sure that all the docker images are deleted from your system or use the command from cert-manager script.